net/shadowsocks-libev/files/shadowsocks-libev.init

```diff
@@ -65,26 +66,28 @@ ss_xxx() {
 
 	json_init
 	ss_mkjson_${cfgtype}_conf || return
-	json_add_boolean use_syslog 1
-	json_add_boolean ipv6_first "$ipv6_first"
-	json_add_boolean fast_open "$fast_open"
-	json_add_boolean reuse_port "$reuse_port"
-	json_add_boolean no_delay "$no_delay"
-	[ -z "$local_address" ] || json_add_string local_address "$local_address"
-	[ -z "$local_port" ] || json_add_int local_port "$local_port"
-	[ -z "$mode" ] || json_add_string mode "$mode"
-	[ -z "$mtu" ] || json_add_int mtu "$mtu"
-	[ -z "$timeout" ] || json_add_int timeout "$timeout"
-	[ -z "$user" ] || json_add_string user "$user"
-	json_dump -i >"$confjson"
-
-	procd_open_instance "$cfgtype.$cfg"
-	procd_set_param command "$bin" -c "$confjson"
-	[ "$verbose" = 0 ] || procd_append_param command -v
-	[ -z "$bind_address" ] || procd_append_param command -b "$bind_address"
-	procd_set_param file "$confjson"
-	procd_set_param respawn
-	procd_close_instance
+	if [ -z "${skip_instance}" ]; then
+		json_add_boolean use_syslog 1
+		json_add_boolean ipv6_first "$ipv6_first"
+		json_add_boolean fast_open "$fast_open"
+		json_add_boolean reuse_port "$reuse_port"
+		json_add_boolean no_delay "$no_delay"
+		[ -z "$local_address" ] || json_add_string local_address "$local_address"
+		[ -z "$local_port" ] || json_add_int local_port "$local_port"
+		[ -z "$mode" ] || json_add_string mode "$mode"
+		[ -z "$mtu" ] || json_add_int mtu "$mtu"
+		[ -z "$timeout" ] || json_add_int timeout "$timeout"
+		[ -z "$user" ] || json_add_string user "$user"
+		json_dump -i >"$confjson"
+
+		procd_open_instance "$cfgtype.$cfg"
+		procd_set_param command "$bin" -c "$confjson"
+		[ "$verbose" = 0 ] || procd_append_param command -v
+		[ -z "$bind_address" ] || procd_append_param command -b "$bind_address"
+		procd_set_param file "$confjson"
+		procd_set_param respawn
+		procd_close_instance
```

Comment:
This indented hunk is idempotent.  procd will compare current and to-be-applied specs of instances, do nothing if they match.  I am afraid adding a new knob is not worth the sweat.